src/Spaceport/Commands/InitCommand.php

```diff
@@ -158,82 +93,15 @@ private function findPHPSettings($ask = true)
     private function askElasticVersion($ask = true)
     {
         if ($ask) {
-            $this->shuttle->setElasticsearchVersion($this->io->choice('What version of Elasticsearch do you need?', ['6', '5', '2'], '5'));
-        }
-    }
-
-    private function askNodeVersion($ask = true)
-    {
-        if ($ask) {
-            $this->shuttle->setNodeVersion($this->io->choice('What version of Node do you need?', ['8', '7', '6.9'], '8'));
+            $this->shuttle->setElasticsearchVersion($this->io->choice('What version of Elasticsearch do you need?', ['6'], '6'));
```

Comment:
Maybe we can expand this and first ask if we need elasticsearch? Apache and PHP are a given but we do not always need ES. Micro optimisation :) And maybe we can make is flexable that if we only support 1 version it does not ask which version but is we add more to the supported list that it will ask.